llvm/unittests/Transforms/Vectorize/VPlanTest.cpp

```diff
@@ -929,7 +929,7 @@ TEST_F(VPRecipeTest, CastVPWidenRecipeToVPUser) {
   SmallVector<VPValue *, 2> Args;
   Args.push_back(Op1);
   Args.push_back(Op2);
-  VPWidenRecipe WidenR(*AI, make_range(Args.begin(), Args.end()));
+  VPWidenRecipe WidenR(*AI, make_range(Args.begin(), Args.end()), {});
```

Comment:
Here and below newly created Instructions are wrapped in widen recipes, should their metadata be preserved (by default) rather than dropped (explicitly)?

Reply:
?

Reply:
This is just for testing casting recipes, so I don't think the metadata matters here